metadata.py

```diff
@@ -28,11 +28,22 @@ def appmetadata() -> AppMetadata:
     )
 
     metadata.add_input(DocumentTypes.VideoDocument, required=True)
-    metadata.add_output(AnnotationTypes.TimeFrame, timeUnit='milliseconds')
-    
+    metadata.add_output(AnnotationTypes.TimeFrame, frameType='bars')
+    metadata.add_output(AnnotationTypes.TimeFrame, frameType='slate')
+    metadata.add_output(AnnotationTypes.TimeFrame, frameType='chyron')
+    metadata.add_output(AnnotationTypes.TimeFrame, frameType='credits')
+
```

Comment:
there are a few problems in the metadata.py

1. output spec doesn't say anything about the `TimePoint`
2. there must be some sort of documentation for the "custom" properties under the `TimePoint` (`label`, `labels`, and `scores`) and `TF` ('score`, `scores`, and `representatives`) since they are not defined in the vocab pages. 
3. `timeUnit` property is missing

Reply:
Done.

For the metadata I assumed that since you added

```python
metadata.add_output(AnnotationTypes.TimeFrame, frameType='bars')
```

the timeUnit was now baked into the new clams-python and therefore not needed anymore, I take it that is not the case?

Reply:
that is not the case yet, unfortunately. That was my bed.